src/components/Wall/Wall.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import '../../App';
+
+
+function Wall(props) {
+    const handleMouseEnter = () => {
+        props.onWallHover(props.id, props.orientation, true);
+    };
+
+    const handleMouseLeave = () => {
+        props.onWallHover(props.id, props.orientation, false);
+    };
+
+    const className = `Wall ${props.orientation} ${props.isHovered ? 'hovered' : ''} ${props.isClicked ? 'clicked' : ''}`;
```

Comment:
This is a good technique but not a regular usage. I know if you write it in 

className={`Wall ${orientation} ${isHovered ? 'hovered' : ''} ${isClicked ? 'clicked' : ''}`}


it will be unreadable but when you pass tailwind-css in future you will understand why I said like that.